### DISCUSSION TITLE
Disable delete in list view for read only

### DISCUSSION THREAD

**Discussion by jymden** [statamic-rad-pack/runway#180] [Help]
Maybe I'm misunderstanding this. But if I set a resource to  'read_only' => true, I can still delete them when looking at them in bulk. The way I found this was this:

* Resource 1 has-many of Resource 2
* I click one of the related models in the listview of Resource 1. 
* I then get to a page where I would have been able to edit it if it was not read only. 
* From there I can then follow the breadcrumbs back to a listing of all Resource 2. 
* There I may still delete any that I have checked. 

The Actions/DeleteModel visibleToBulk() function should probably be set to false if the resource is set to read only. If I'm not misunderstanding the behavior of this, I will try to find a fix and make a PR. But please let me know if I'm in the wrong.

**Comment by duncanmcclean** [comment]
Ah yes, doing it in bulk should respect the ‘read only’ setting (because it does individually)

**Comment by jymden** [comment]
Man. I'm really on your back today :-). It's great with such quick responses. Thank you. I could look into it.

**Comment by jymden** [comment]
I made PR for a quick fix that seems to do the job.
